src/Turbo/README.md

```diff
@@ -0,0 +1,626 @@
+
+```php
+<?php
+
+namespace App\Entity;
+
+use Symfony\UX\Turbo\Broadcast;
+
+#[Broadcast(transports: ['hub1', 'hub2'])]
+/** ... */
+class Book
+{
+    // ...
+}
+```
+
+Finally, generate the HTML attributes registering the Stimulus controller
+corresponding to your transport by passing an extra argument to `turbo_stream_listen()`:
+
+```twig
+<div id="messages" {{ turbo_stream_listen('App\Entity\Book', 'hub2') }}></div>
+```
+
+### Registering a Custom Transport
+
+If you prefer using another protocol than Mercure, you can create your custom transports:
+
+```php
+<?php
+
+namespace App\Turbo;
+
+use Symfony\UX\Turbo\Broadcast;
+use Symfony\UX\Turbo\Broadcaster\BroadcasterInterface;
+
+class Broadcaster implements BroadcasterInterface
+{
+    public function broadcast(object $entity, string $action): void
+    {
+        // This method will be called everytime an object marked with the #[Broadcast] attribute is changed
+        $attribute = (new \ReflectionClass($entity))->getAttributes(Broadcast::class)[0] ?? null;
+        // ...
+    }
+}
+```
+
+```php
+<?php
+
+namespace App\Turbo;
+
+use Symfony\UX\Turbo\Twig\TurboStreamListenRendererInterface;use Symfony\WebpackEncoreBundle\Twig\StimulusTwigExtension;use Twig\Environment;
+
+class TurboStreamListenRenderer implements TurboStreamListenRendererInterface
+{
+    public function __construct(
+        private StimulusTwigExtension $stimulusTwigExtension,
+    ) {}
+
+    /**
+     * @param string|object $topic
+     */
+    public function renderTurboStreamListen(Environment $env, $topic) : string{
+        $this->stimulusTwigExtension->renderStimulusController($env, 'your_stimulus_controller', [/* controller values such as topic */]);
+    }
+
+    public static function getDefaultIndexName(): string
+    {
+        // The transport name, used as key in the service locator
+        // Alternatively, configure it using the other supported methods: https://symfony.com/doc/current/service_container/service_subscribers_locators.html#indexing-the-collection-of-services
+        return 'my-transport';
+    }
+}
+```
+
+Because these classes implement the `BroadcasterInterface` and `TurboStreamListenRendererInterface` interfaces,
+the related services will be [autoconfigured](https://symfony.com/doc/current/service_container.html#the-autoconfigure-option).
+
+## Backward Compatibility promise
+
+This bundle aims at following the same Backward Compatibility promise as the Symfony framework:
+[https://symfony.com/doc/current/contributing/code/bc.html](https://symfony.com/doc/current/contributing/code/bc.html)
+
+However, it is currently considered
+[**experimental**](https://symfony.com/doc/current/contributing/code/experimental.html),
+meaning it is not bound to Symfony's BC policy for the moment.
+
+## Credits
+
+Created by [Kévin Dunglas](https://dunglas.fr). Sponsored by [Les-Tilleuls.coop](https://les-tilleuls.coop).
```

Comment:
Can you please remove this line? Symfony doesn't put such credits anywhere, it's a community project :)